### ISSUE TITLE
Add script to update sidecar/cephcsi version in all required files.

### ISSUE THREAD

**Issue by Madhu-1** [ceph/ceph-csi#3171]
Based on the version specified in the build.env we need a script to update all the required files. this helps to reduce the burden. we don't need to worry about missing updates in some places. Currently, it's difficult to track which files need updates when a version is updated.

**Comment by humblec** [comment]
Agree.. let me try to push something which can help here.. taking this issue

**Comment by Madhu-1** [comment]
Thanks @humblec

**Comment by nixpanic** [comment]
Maybe dependabot can be configured to check for updated container images, somehow?
